src/Authentication/Authenticators/Session.php

```diff
@@ -922,17 +922,17 @@ protected function rememberUser(User $user): void
             $user,
             $selector,
             $this->hashValidator($validator),
-            $expires
+            $expires->format('Y-m-d H:i:s')
```

Comment:
why !? :monocle_face:

Reply:
Ah, I saw @TODO.

Reply:
Yes, we must pass a string. Otherwise it will be a breaking change.